pyproject.toml

```diff
@@ -12,6 +12,7 @@ requests = "^2.31.0"
 pytest = "^8.2.2"
 pymongo = "^4.9.1"
 mongomock = "^4.2.0.post1"
+safety = "^3.2.8"
```

Comment:
Minor, but this is a nice prompt to spot that this could be a dev dependency, meaning that our prod image has one less library that it has to worry about. Probably not worth changing here but if you're interested then you can [read about how that's suggested in the docs](https://python-poetry.org/docs/managing-dependencies/#dependency-groups)